Updated ReadMe

sdhamani opened the pull request.

sdhamani committed:
Updated ReadMe

vvaibhavdesai commented:
This looks good

sdhamani committed:
added iframe class to Image component

sdhamani committed:
Added width condition for buttons

sdhamani committed:
made iframes responsive

sdhamani commented:
Hello,
Live Website link: https://serene-ui.netlify.app/
Please feel free to go through it
Thank You.

purvasheth commented:
![image](https://user-images.githubusercontent.com/31120070/112639001-d3baae00-8e65-11eb-9a2c-55c0041e0b41.png)


Add some margin/padding to `Getting Started`

purvasheth commented on app.js:
```diff
@@ -53,9 +33,13 @@ btnTScrollUp.addEventListener("click", function () {
-  menuitem.style.display = "none";
-  myFunctiontoggle();
+  console.log("Ins");
+  console.log(screenWidth);
```
always remove `console.logs` from final code

sdhamani replied:
Removed them. 
I Will keep this in mind.
Thanks.

purvasheth commented on styles.css:
```diff
@@ -442,14 +430,6 @@ Alerts  */
-} */
-
 .card-image {
   width: 100%;
```
I think `max-width:100%` kiya toh `height:auto` is not necessary. Try it out!

sdhamani replied:
Ya, I guess that is surplus. Removed it too.

purvasheth commented on styles.css:
```diff
@@ -468,15 +448,13 @@ Alerts  */
 }
 
 .card-dismiss-btn:hover {
   padding: 10px 15px;
```
Use `rem` even for padding. Makes them more consistent.

sdhamani replied:
Yes, will avoid using px as suggested by Kushagra too.

sdhamani commented:
Hello Purva ,
As discussed, it is a browser-specific issue so we can ignore it.
![image](https://user-images.githubusercontent.com/74397480/112640198-1e88f580-8e67-11eb-94e2-c560f72131e7.png)
Thanks for the review.

purvasheth reviewed (commented):
Hi @sdhamani,

Accha banaya hai. 
Naam bhi mast hai -"SERENE". Library dekhke ekdum "serene" laga. :)
Thode minor code-level comments bhi hai. Dekh lena ek baar.
Thode aur suggestions 
 -  make `border:transparent` or `border: var(primary-color)` for primary buttons.
 - primary and secondary buttons seem same. Make the secondary button different.

Akshaay9 reviewed (commented):
Hello
The component Library Looks good
One suggestion
1) add ` Cursor:pointer` to left side of Navigation Bar (on List items)
2) Add some padding on Top and botoom of alert bar

sdhamani commented:
> Hello
> The component Library Looks good
> One suggestion
> add ` Cursor:pointer` to left side of Navigation Bar (on List items)

Thanks, Akshay for reviewing the PR.
I will add the cursor to the items in left Navigation Bar .

Akshaay9 reviewed (approved):
1) on `badge-on-icon-class`
add this code
`
.badge-on-icon {
    height: 1.4rem;
    border-radius: 50%;
    position: absolute;
    top: 0px;
    left: 3.5rem;
    background-color: var(--primary-color);
    color: var(--button-text-color);
    font-size: 0.8rem;
    display: flex;
    justify-content: center;
    align-items: center;
    padding: 0.4rem;
`
It will make it look bigger and better

Akshaay9 reviewed (commented):
FInal review -)
on buttons add
`Outline:none`
it removes unnecessary border/outline.
Your Component library Looks Really good
Great Job..

sdhamani committed:
Changed styles for badge-on-icon class

sdhamani commented:
> 1. on `badge-on-icon-class`
>    add this code
>    `.badge-on-icon { width: -39.7rem; height: 1.4rem; border-radius: 50%; position: absolute; top: 0px; left: 3.5rem; background-color: var(--primary-color); color: var(--button-text-color); font-size: 0.8rem; display: flex; justify-content: center; align-items: center; padding: 0.4rem;`
>    It will make it look bigger and better

Ya added these styles and now it really looks impressive.
Thanks a lot. :)

sdhamani committed:
removed border style from buttons

sdhamani commented:
> FInal review -)
> on buttons add
> `Outline:none`
> it removes unnecessary border/outline.
> Your Component library Looks Really good
> Great Job..

Thanks a lot, Akshay  :).
As discussed , I added border: none to buttons and it worked

sdhamani committed:
Added padding to alerts

ishanjirety commented:
![image](https://user-images.githubusercontent.com/62433518/112673644-fad7a680-8e8a-11eb-9512-c9db6f24545b.png)

Avoid using var. Instead, you should use const or let.

sdhamani committed:
Changed keyword from var to const

sdhamani committed:
Changed footer positioning

sdhamani committed:
footer display changed

sdhamani commented:
> ![image](https://user-images.githubusercontent.com/62433518/112673644-fad7a680-8e8a-11eb-9512-c9db6f24545b.png)
> 
> Avoid using var. Instead, you should use const or let.

Hi Ishan,
I have changed the keywords to let and const.
Thanks for reviewing.

ishanjirety commented:
The code looks really good, the library is working properly. 
I think it's good to go.

sdhamani commented:
> The code looks really good, the library is working properly.
> I think it's good to go.

Thanks Captain :)

AnkitKarnAK commented:
Hey Sagar
1. Your primary & secondary button looks same unless user hovers on it. They should be different on first look instead of just on hover.
2. Links in footer have very less left-right margin space. moreover, you gave space between `Git Hub` & it looks like four different links.
![Annotation 2021-03-29 024529](https://user-images.githubusercontent.com/45123508/112768265-e1745d00-9038-11eb-829e-0c90afd2b9c2.png)

sdhamani commented:
> Hey Sagar
> 
> 1. Your primary & secondary button looks same unless user hovers on it. They should be different on first look instead of just on hover.
> 2. Links in footer have very less left-right margin space. moreover, you gave space between `Git Hub` & it looks like four different links.
>    ![Annotation 2021-03-29 024529](https://user-images.githubusercontent.com/45123508/112768265-e1745d00-9038-11eb-829e-0c90afd2b9c2.png)

Hi Ankit,
Thanks for reviewing.
Changed bg color of my secondary button and added margin to the footer too.
let me know if any changes are needed.
